src/components/buttonsGroup/FilterButtonsGroup.tsx

```diff
@@ -0,0 +1,55 @@
+import { Stack, Typography } from '@mui/material'
+import { SecondaryBtn } from '@/components/buttons/SecondaryBtn'
+import { ReactNode } from 'react'
+
+export type FilterButtons = {
+  name: string
+  onClick: (index: number) => void
+}
+
+const FilterButtonGroup = ({
+  filterButtons,
+  activeButtonIndex,
+}: {
+  filterButtons: FilterButtons[]
+  activeButtonIndex: number | undefined
+}) => {
+  return (
+    <Stack
+      sx={(theme) => ({
+        border: `1px solid ${theme.color.borders.border}`,
+        borderRadius: 1,
+        display: 'flex',
+        gap: '12px',
+        alignItems: 'center',
+        padding: '4px',
+      })}
+      direction={'row'}
+    >
+      {filterButtons.map((item, index) => {
+        return (
+          <SecondaryBtn
+            key={index}
```

Comment:
Is there something other than `index` we can use for a key? Using indexes as keys is typically not recommended.